packages/components/socialFeed/SocialCard/cards/SocialArticleMarkdownCard.tsx

```diff
@@ -0,0 +1,197 @@
+import { LinearGradient } from "expo-linear-gradient";
+import React, { FC, memo, useEffect, useState } from "react";
+import { StyleProp, useWindowDimensions, View, ViewStyle } from "react-native";
+
+import { BrandText } from "../../../BrandText";
+import { OptimizedImage } from "../../../OptimizedImage";
+import { CustomPressable } from "../../../buttons/CustomPressable";
+import { SpacerColumn } from "../../../spacer";
+import { FlaggedCardFooter } from "../FlaggedCardFooter";
+import { SocialCardFooter } from "../SocialCardFooter";
+import { SocialCardHeader } from "../SocialCardHeader";
+import { SocialCardWrapper } from "../SocialCardWrapper";
+
+import { Post } from "@/api/feed/v1/feed";
+import defaultThumbnailImage from "@/assets/default-images/default-article-thumbnail.png";
+import { useAppNavigation } from "@/hooks/navigation/useAppNavigation";
+import { zodTryParseJSON } from "@/utils/sanitize";
+import {
+  ARTICLE_THUMBNAIL_IMAGE_MAX_WIDTH,
+  SOCIAl_CARD_BORDER_RADIUS,
+} from "@/utils/social-feed";
+import {
+  neutral00,
+  neutral33,
+  neutralA3,
+  withAlpha,
+} from "@/utils/style/colors";
+import { fontRegular13, fontRegular15 } from "@/utils/style/fonts";
+import {
+  layout,
+  RESPONSIVE_BREAKPOINT_S,
+  SOCIAL_FEED_BREAKPOINT_M,
+} from "@/utils/style/layout";
+import { ZodSocialFeedArticleMetadata } from "@/utils/types/feed";
+
+const ARTICLE_CARD_PADDING_VERTICAL = layout.spacing_x2;
+const ARTICLE_CARD_PADDING_HORIZONTAL = layout.spacing_x2_5;
+
+export const SocialArticleMarkdownCard: FC<{
+  post: Post;
+  isPostConsultation?: boolean;
+  style?: StyleProp<ViewStyle>;
+  refetchFeed?: () => Promise<any>;
+  isFlagged?: boolean;
+  disabled?: boolean;
+}> = memo(
+  ({ post, isPostConsultation, refetchFeed, style, isFlagged, disabled }) => {
+    const navigation = useAppNavigation();
+    const [localPost, setLocalPost] = useState<Post>(post);
+    const [viewWidth, setViewWidth] = useState(0);
+    const { width: windowWidth } = useWindowDimensions();
+
+    const articleCardHeight =
+      windowWidth < SOCIAL_FEED_BREAKPOINT_M ? 214 : 254;
+    const thumbnailImageWidth = viewWidth / 3;
+    const borderRadius =
+      windowWidth < RESPONSIVE_BREAKPOINT_S ? 0 : SOCIAl_CARD_BORDER_RADIUS;
+
+    const metadata = zodTryParseJSON(
+      ZodSocialFeedArticleMetadata,
```

Comment:
we shouldn't reuse the old type for the new article kind IMO

Reply:
I agree, but we didn't discuss about the new shape. I don't want to commit, then fix the review, then commit, then fix the review, etc. We need to status on that once

Reply:
https://github.com/TERITORI/teritori-dapp/pull/1451/commits/0aae56913199797f69a1cc7bc0f82d17cd419792